### ISSUE TITLE
prefilled templates

### ISSUE THREAD

**Issue by jeremybreeschoten** [PrivateBin/PrivateBin#721]
## Steps to reproduce

1.Nothing to reproduce

### What should happen
Is there a way to add kind of prefilled templates to this? For example if we send user information, we can select something and then have all information prefilled?

**Comment by elrido** [comment]
I don't think this is a very common feature that should be introduced into the generic web UI.

That said, you could use any of the [third party clients](https://github.com/PrivateBin/PrivateBin/wiki/Third-party-clients) against your instance to set up such automation. You could write a little html form in PHP for your dynamic inputs and on post submit these into a templated text using the PHP library. Or if your more comfortable with shell scripting, you'd template some text file with shell utilities and submit it via one of the CLI clients - some allow reading the paste from STDIN, so you could sed-replace some things in a text file and pipe it into the client.
